src/containers/Flow/FlowList/FlowList.tsx

```diff
@@ -35,7 +35,9 @@ const getName = (text: string, keywordsList: any, roles: any) => {
     <div className={styles.NameText}>
       {text}
       <br />
-      <span className={styles.Keyword}>{keywords.join(', ')}</span>
+      <span className={styles.Keyword}>
```

Comment:
It might be better to restrict this based on the string length that's rendered rather than the number of words.